README.md

```diff
@@ -300,31 +311,39 @@ This section provides a step-by-step procedure for deploying the resource aggreg
       $ sudo logrotate -v -f /etc/logrotate.d/odimra
   	  ```
   
-11. Default Credentails
 
-	```
-	ODIMRA:
-	username: admin
-	password: Od!m12$4
+11. To add the Generic Redfish Plugin and servers to the resource aggregator for ODIM, refer to the following readme.  
+    https://github.com/ODIM-Project/ODIM/blob/development/svc-aggregation/README.md
+	
+	
+### Default user credentials for ODIMRA and GRF Plugin 
 
-	GRF PLUGIN:
-	username: admin
-	password: GRFPlug!n12$4
-	``` 
 
-12. Refer to the following readme to add the Generic Redfish Plugin and then add the servers to ODIMRA.  
-    https://github.com/ODIM-Project/ODIM/blob/development/svc-aggregation/README.md
-    
+ODIMRA:
+
+```
+username: admin
+password: Od!m12$4
+```
+
+GRF PLUGIN:
+
+```
+username: admin
+password: GRFPlug!n12$4
+``` 
+ 
+ 
   
-  #  Modifying default configuration parameters for the resource aggregator
+#  Modifying default configuration parameters for the resource aggregator
 
-1.   Navigate to the `odim_1` container using the following command: 
+1.   Navigate to the `odimra_1` container using the following command: 
 
       ```
-     $ docker exec -it odim_1 /bin/bash
+     $ docker exec -it odimra_1/bin/bash
```

Comment:
docker exec -it odimra_1/bin/bash

the command is wrong, Please change it

correct command:  docker exec -it build_odimra_1 /bin/bash

Reply:
Changed to the right command.

Reply:
Space is missing after 1